src/meilisearch/mod.rs

```diff
@@ -0,0 +1,312 @@
+use std::{borrow::Cow, collections::HashMap};
+
+use testcontainers::{
+    core::{wait::HttpWaitStrategy, ContainerPort, WaitFor},
+    Image,
+};
+
+const NAME: &str = "getmeili/meilisearch";
+const TAG: &str = "v1.8.3";
+const MEILISEARCH_PORT: ContainerPort = ContainerPort::Tcp(7700);
+
+/// Module to work with [`Meilisearch`] inside of tests.
+///
+/// Starts an instance of Meilisearch.
+/// This module is based on the official [`Meilisearch docker image`] documented in the [`Meilisearch docker docs`].
+///
+/// # Example
+/// ```
+/// use testcontainers_modules::{meilisearch, testcontainers::runners::SyncRunner};
+///
+/// let meilisearch_instance = meilisearch::Meilisearch::default().start().unwrap();
+///
+/// let dashboard = format!(
+///     "http://{}:{}",
+///     meilisearch_instance.get_host().unwrap(),
+///     meilisearch_instance.get_host_port_ipv4(7700).unwrap()
+/// );
+/// ```
+///
+/// [`Meilisearch`]: https://www.meilisearch.com/
+/// [`Meilisearch docker docs`]: https://www.meilisearch.com/docs/guides/misc/docker
+/// [`Meilisearch docker image`]: https://hub.docker.com/_/getmeili/meilisearch
+#[derive(Debug)]
```

Comment:
Some derive `Clone` here. Should this be added?

```suggestion
#[derive(Debug, Clone)]
```

Reply:
It doesn’t bring any issues for us, but I think it might be useful in some cases, so let’s add it. 

Thank you!